staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss_cache_test.go

```diff
@@ -95,18 +95,18 @@ func TestVMSSVMCache(t *testing.T) {
 	// validate deleteCacheForNode().
 	vm := virtualMachines[0]
 	vmName := to.String(vm.OsProfile.ComputerName)
-	err = ss.deleteCacheForNode(vmName)
+	err = ss.deleteCacheForNode(vmName, allowUnsafeRead)
```

Comment:
Why do we have  `allowUnsafeRead ` on `deleteXXX` funcs?

Reply:
`deleteCacheForNode` is removing the vm entry for the node from cache. As long as the entry exists (expired/ unexpired) we want to delete it from cache so we can update it in the next event that triggers us.